lib/react.dart

```diff
@@ -126,7 +147,7 @@ abstract class Component {
   /// Private reference to the value of [context] for the upcoming render cycle.
   ///
   /// Useful for ReactJS lifecycle methods [shouldComponentUpdateWithContext] and [componentWillUpdateWithContext].
-  Map nextContext;
+  dynamic nextContext;
```

Comment:
For my own knowledge - what are the possible expected types?

Reply:
literally anything... hence dynamic. lol but more often than not probably like `String`, `List`, `Map`, `int`, `bool`